tests/test_viz.py

```diff
@@ -98,7 +97,8 @@ def test_dense_input_line():
 
     fig.savefig("test_model_line.png")
 
-    dg = DenseGraph(model)
+    dg = DenseGraph(model, input_color='#FF0000', inner_color='#00FF00', output_color='#FFFF00',
```

Comment:
same here please don't change the existing test since all your parameters are optional.